### PR TITLE
update: FAQ to give more info about cluster migrations

### PR DESCRIPTION
**Pull request by benjaminch**
Ticket: ENG-1468

### CODE REVIEW

File: website/docs/useful-resources/faq.md.erb [hunk 1]
```diff
@@ -53,6 +53,20 @@ You have to create N CNAME, one per custom domain
 
 The Qovery team manages your Kubernetes cluster's upgrade, and you don't have to think about it. Upgrades from one minor Kubernetes version to another require a good amount of tests to make sure everything goes smoothly with zero interruptions for your app. This is why Qovery always provides 1 or 2 minor versions below the last one offered by the cloud provider. Our goal is to guarantee you the maximum uptime.
 
+### How do you upgrade clusters to new versions?
+
+Usually we work on a given upgrade for one month of intensive testing on our end in order to make sure everything will be smooth for you. Once we are pretty confident our stack is stable, we move on with the following steps wich last approximatively 3 weeks:
```
Comment (acarranoqovery) [inline website/docs/useful-resources/faq.md.erb:58]: maybe we should add all this info in the Cluster doc and add links in the FAQ there https://hub.qovery.com/docs/using-qovery/configuration/clusters/#how-does-qovery-handle-cluster-updates-and-upgrades

Comment (benjaminch) [reply website/docs/useful-resources/faq.md.erb]: Updated

**Review by acarranoqovery — APPROVED** [review]
